plugins/woocommerce/changelog/fix-49980-overlay-navigation-title-icon

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: update
+
+Update: New title and icon for Overlay Navigation block
```

Comment:
I think we just need a comment here so the changelog doesn't make it into the official list.

Reply:
Good call! I updated the changelog to comment. Thanks!